### PR TITLE
Revamp Google Sheets Connector

### PR DESCRIPTION
**Pull request by DimuthuMadushan**
## Purpose

Fixes: [#6063](https://github.com/ballerina-platform/ballerina-library/issues/6063)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the specification~
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: examples/grid-filtering/.github/README.md
Comment (ThisaruGuruge) [inline examples/grid-filtering/.github/README.md:1]: Let's use the example name for the MD file name and add a symlink to the README file.

**Review by ThisaruGuruge — COMMENTED** [review]
Let's rename the connector to `googleapis.gsheets` and bump a major version.

File: README.md [hunk 1]
```diff
@@ -1,55 +1,231 @@
...
+
+    <img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-googleapis.sheets/main/docs/setup/resources/exchange-tokens.png alt="Exchange Tokens" width="50%">
+
+## Quickstart
+
+To use the `Google Sheets` connector in your Ballerina project, modify the `.bal` file as follows:
+
+### Step 1: Import connector
+
+Import the `ballerinax/googleapis.sheets` module.
```
Comment (aashikam) [inline README.md:87]: I thought this was renamed to `gsheets`?

Comment (DimuthuMadushan) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.sheets/pull/289/commits/3dbed6c101ee7ef85f0775417794ba30e9a6e03f

File: ballerina/tests/data-generator/data-generator.bal [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import ballerina/io;
+import ballerina/log;
```
Comment (ThisaruGuruge) [inline ballerina/tests/data-generator/data-generator.bal:2]: Let's move these import statements after the license header.

File: ballerina/utils.bal [hunk 3]
```diff
@@ -19,144 +19,85 @@ import ballerina/jballerina.java as java;
...
+    json payload = check response.getTextPayload();
+    return error(payload.toString());
 }
 
 # Create a random UUID removing the unnecessary hyphens which will interrupt querying opearations.
-# 
+#
 # + return - A string UUID without hyphens
 public function createRandomUUIDWithoutHyphens() returns string {
     string? stringUUID = java:toString(createRandomUUID());
```
Comment (ThisaruGuruge) [inline ballerina/utils.bal:90]: Check similar occurences.

```suggestion
    string? stringUuid = java:toString(createRandomUUID());
```

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**